VideoLocker/src/main/java/org/edx/mobile/view/LoginActivity.java

```diff
@@ -268,24 +268,15 @@ public void showNoNetworkDialog() {
     }
 
     @Override
-    public boolean showErrorMessage(String header, String message, boolean isPersistent) {
+    public boolean showErrorMessage(String header, String message) {
         if (message != null) {
-            return super.showErrorMessage(header, message, isPersistent);
+            return super.showErrorMessage(header, message);
         } else {
-            return super.showErrorMessage(header, getString(R.string.login_failed), isPersistent);
+            return super.showErrorMessage(header, getString(R.string.login_failed));
         }
     }
 
     @Override
-    public boolean dispatchTouchEvent(MotionEvent ev) {
-        Animation errorMessageAnim = activityLoginBinding.errorLayout.getAnimation();
-        if (errorMessageAnim == null || errorMessageAnim.hasEnded()) {
-            ViewAnimationUtil.hideMessageBar(activityLoginBinding.errorLayout);
-        }
-        return super.dispatchTouchEvent(ev);
-    }
-
-    @Override
     protected void onOnline() {
         super.onOnline();
         hideErrorMessage();
```

Comment:
Hmm, actually, onOnline and onOffline are never called, so I guess that won't happen.

Weird